lib/take_away.rb

```diff
@@ -0,0 +1,54 @@
+
+class TakeAway
+  attr_reader :order, :dish_hash
+  def initialize
+    @order = {}
+    @dish_hash = {}
+    #create_dish_hash
+  end
+
+  def show_menu
+    # can be refactored to take menu.csv as input file
+    filename = "lib/menu.txt"
+    File.open filename do |f|
+      f.each { |line| puts line }
+    end
+  end
+
+  def enter_order
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for enter_order is too high. [17.52/15] (http://c2.com/cgi/wiki?AbcMetric)<br>Metrics/MethodLength: Method has too many lines. [13/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)